Dockerfile

```diff
@@ -13,6 +13,9 @@ RUN yum install -y python-requests && \
 RUN mkdir /tftpboot && \
     cp /usr/share/ipxe/undionly.kpxe /usr/share/ipxe/ipxe.efi /tftpboot/
 
+# Required by vendor driver Fujitsu iRMC
+RUN pip install python-scciclient
```

Comment:
Please no pip, we're removing EPEL from here. scciclient should be available from the repository we use.